ports/mimxrt/boards/MIMXRT1010_EVK/mpconfigboard.h

```diff
@@ -8,3 +8,6 @@
 #define MICROPY_HW_LED_ON(pin) (mp_hal_pin_high(pin))
 #define MICROPY_HW_LED_OFF(pin) (mp_hal_pin_low(pin))
 #define BOARD_FLASH_CONFIG_HEADER_H "evkmimxrt1010_flexspi_nor_config.h"
+
+#define MICROPY_BOARD_ROOT_POINTERS \
+    void *machine_pin_irq_objects[2 * 32];
```

Comment:
would be simpler for a board to define the number of objects, eg `#define MICROPY_HW_NUM_PIN_IRQS (2 * 32)`, and then move this definition into `MICROPY_PORT_ROOT_POINTERS` in `mpconfigport.h`

Reply:
Yes, indeed.